build/_k-standard-goals.shade

```diff
@@ -118,6 +110,38 @@ default DNX_TOOLS_FEED = 'https://www.myget.org/F/dnxtools/api/v2'
     }
   }
 
+#build-compile target='compile' if='IsBuildV2'
```

Comment:
I just say it because I often run `build compile` to get nupkgs as quickly as possible.